packages/docprovider/src/awareness.ts

```diff
@@ -0,0 +1,104 @@
+// From https://en.wikipedia.org/wiki/Moons_of_Jupiter
+export const moonsOfJupyter = [
+  'Metis',
+  'Adrastea',
+  'Amalthea',
+  'Thebe',
+  'Io',
+  'Europa',
+  'Ganymede',
+  'Callisto',
+  'Themisto',
+  'Leda',
+  'Ersa',
+  'Pandia',
+  'Himalia',
+  'Lysithea',
+  'Elara',
+  'Dia',
+  'Carpo',
+  'Valetudo',
+  'Euporie',
+  'Eupheme',
+  // 'S/2003 J 18',
+  // 'S/2010 J 2',
+  'Helike',
+  // 'S/2003 J 16',
+  // 'S/2003 J 2',
+  'Euanthe',
+  // 'S/2017 J 7',
+  'Hermippe',
+  'Praxidike',
+  'Thyone',
+  'Thelxinoe',
+  // 'S/2017 J 3',
+  'Ananke',
+  'Mneme',
+  // 'S/2016 J 1',
+  'Orthosie',
+  'Harpalyke',
+  'Iocaste',
+  // 'S/2017 J 9',
+  // 'S/2003 J 12',
+  // 'S/2003 J 4',
+  'Erinome',
+  'Aitne',
+  'Herse',
+  'Taygete',
+  // 'S/2017 J 2',
+  // 'S/2017 J 6',
+  'Eukelade',
+  'Carme',
+  // 'S/2003 J 19',
+  'Isonoe',
+  // 'S/2003 J 10',
+  'Autonoe',
+  'Philophrosyne',
+  'Cyllene',
+  'Pasithee',
+  // 'S/2010 J 1',
+  'Pasiphae',
+  'Sponde',
+  // 'S/2017 J 8',
+  'Eurydome',
+  // 'S/2017 J 5',
+  'Kalyke',
+  'Hegemone',
+  'Kale',
+  'Kallichore',
+  // 'S/2011 J 1',
+  // 'S/2017 J 1',
+  'Chaldene',
+  'Arche',
+  'Eirene',
+  'Kore',
+  // 'S/2011 J 2',
+  // 'S/2003 J 9',
+  'Megaclite',
+  'Aoede',
+  // 'S/2003 J 23',
+  'Callirrhoe',
+  'Sinope'
+];
+
+/**
+ * Get a random user-name based on the moons of Jupyter.
+ * This function retuns names like "Anonymous Io" or "Anonymous Metis".
+ */
+export const getAnonymousUserName = (): string =>
+  'Anonymous ' +
+  moonsOfJupyter[Math.floor(Math.random() * moonsOfJupyter.length)];
+
+export const usercolors = [
+  '#30bced',
+  '#6eeb83',
+  '#ffbc42',
+  '#ecd444',
+  '#ee6352',
+  '#9ac2c9',
+  '#8acb88',
+  '#1be7ff'
```

Comment:
Assuming font-style **bold** and white text (as it currently is) I used [contrast checker](https://webaim.org/resources/contrastchecker/) to adjust the lightness of the suggested colors to the closest value to the current one that meets the 3:1 contrast ratio and WCAG AA standard for large text. We may want to use a higher contrast (4.5:1) or add a subtle shadow given the font size is small.

No change was needed for ee6352 which already had 3.21:1 contrast ratio

```suggestion
  '#12A0D3',
  '#17AB30',
  '#CC8500',
  '#A79011',
  '#ee6352',
  '#609DA9',
  '#4BA749',
  '#00A1B3'
```

Reply:
cross-ref https://github.com/jupyterlab/jupyterlab/issues/9399